Source/Core/Common/Arm64Emitter.h

```diff
@@ -549,11 +550,11 @@ struct LogicalImm
     // pick the next sequence of ones. This ensures we get a complete element
     // that has not been cut-in-half due to rotation across the word boundary.
 
-    const size_t rotation = Common::CountTrailingZeros(value & (value + 1));
+    const int rotation = std::countr_zero(value & (value + 1));
     const u64 normalized = Common::RotateRight(value, rotation);
 
-    const size_t element_size = Common::CountTrailingZeros(normalized & (normalized + 1));
-    const size_t ones = Common::CountTrailingZeros(~normalized);
+    const int element_size = std::countr_zero(normalized & (normalized + 1));
+    const int ones = std::countr_zero(~normalized);
```

Comment:
```suggestion
    const int ones = std::countr_one(normalized);
```

(I think this will compile down to the same machine code anyway, but just for cleanness...)